Makefile

```diff
@@ -25,8 +33,14 @@ GPGME_ENV = CGO_CFLAGS="$(shell gpgme-config --cflags 2>/dev/null)" CGO_LDFLAGS=
 
 all: tools test validate .gitvalidation
 
-build: vendor
-	@$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
+build: vendor build-internal
+
+build-internal:
+	$(GPGME_ENV) go build $(BUILDFLAGS) $(PACKAGES)
+
+cross: vendor
+	GOOS=windows $(MAKE) build-internal BUILDTAGS="$(BUILDTAGS) $(BUILD_TAG_WINDOWS_CROSS)"
+	GOOS=darwin $(MAKE) build-internal BUILDTAGS="$(BUILDTAGS) $(BUILD_TAG_DARWIN_CROSS)"
```

Comment:
(Non-blocking: This is a tiny bit imprecise, basically it works only because with the current set of tags, adding more tags never breaks a build.

Maybe `.travis.yml` should be overriding `PLATFORM_BUILD_TAG` instead of `BUILDTAGS`?

Non-blocking, it’s likely enough that testing this in all the platforms is not really worth the effort, at least right now.)

Reply:
i tried a couple of permutations like that. I don't like the way the travis job is just passing through to a docker container, but the current way runs.